BUILD.gn

```diff
@@ -198,7 +198,10 @@ group("create_dist") {
     # zip will be created along with installer
     deps += [ ":create_symbols_dist" ]
   } else {
-    deps += [ ":create_dist_zips" ]
+    deps += [
+      ":create_dist_zips",
+      ":create_symbols_dist",
+    ]
```

Comment:
nit: How about this to be more readable? :) 
```
  if (is_android) {
    deps += [ "build/android:sign_app" ]
  } else {
    deps += [ ":create_symbols_dist" ]
  }

  if (is_win || is_mac) {
    deps += [ ":create_dist_zips" ]
  }
```

Reply:
Ok, I've changed this section a bit.